packages/ui/src/components/vuestic-components/va-chip/VaChip.vue

```diff
@@ -1,40 +1,44 @@
 <template>
-  <component
-    :is="tagComputed"
-    v-if="valueComputed"
-    :href="hrefComputed"
-    :target="target"
-    :to="to"
-    :replace="replace"
-    :exact="exact"
-    :active-class="activeClass"
-    :exact-active-class="exactActiveClass"
-    class="va-chip"
-    :class="computedClass"
-    :style="computedStyle"
-    :tabindex="indexComputed"
+  <div
```

Comment:
i think this element should be 'inline-flex', like a child element

Reply:
fixed in https://github.com/epicmaxco/vuestic-ui/pull/516/commits/d1245b9d935941edb873b3ba355ba74d65006006